docusaurus/docs/Android/04-compose/04-message-components/01-messages-screen.mdx

```diff
@@ -43,7 +44,7 @@ The `MessagesScreenComponent` exposes two actions, as per the signature:
 fun MessagesScreen(
     ..., // State
     onBackPressed: () -> Unit = {},
```

Comment:
It's not related to this PR, there is a small typo on line 40. `MessagesScreenComponent` should be `MessagesScreen`.

Reply:
Good catch, I'll fix that one :)